pkg/commands/process/settings/rules/ruby/third_parties/clickhouse.yml

```diff
@@ -0,0 +1,38 @@
+patterns:
+  - pattern: |
+      $<ROWS> << $<DATA_TYPE>
+    filters:
+      - variable: ROWS
+        detection: ruby_third_parties_clickhouse_insert_rows
+      - variable: DATA_TYPE
+        detection: datatype
+languages:
+  - ruby
+auxiliary:
+  - id: ruby_third_parties_clickhouse_insert_rows
+    patterns:
+      - |
+        Clickhouse.connection.insert_rows() { |$<!>$<_:identifier>| }
+trigger: local
+severity:
+  default: low
+  PII: critical
+  PHI: medium
+  PD: high
+skip_data_types:
+  - "Unique Identifier"
+metadata:
+  description: "Do not store sensitive data in ClickHouse."
+  remediation_message: |
+    ## Description
+    Leaking sensitive data to a third-party service is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to ClickHouse.
+
+    ## Remediations
+
+    When sending data to third-party services, ensure all sensitive data is removed.
+
+    ## Resources
```

Comment:
I'm going to merge this in the meantime, but ping @markmichon for the description. 

I've re-used the wording from Google Dataflow, although ClickHouse is something a bit different (namely, "a column-oriented SQL database management system (DBMS)") it is still a third-party service, so this general description seems appropriate to me.